.changes/unreleased/tinymce-TINY-10642-2024-02-29.yaml

```diff
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Removed
+body: Removed deprecated things
```

Comment:
We need to describe in more detail what we removed here maybe multiple log items even.